public/app/features/alerting/unified/mocks/server/configure.ts

```diff
@@ -60,20 +58,6 @@ export const setFolderResponse = (response: Partial<FolderDTO>) => {
   server.use(handler);
 };
 
-/**
- * Makes the mock server respond with different Grafana Alertmanager config
- */
-export const setGrafanaAlertmanagerConfig = (config: AlertManagerCortexConfig) => {
-  server.use(getGrafanaAlertmanagerConfigHandler(config));
-};
-
-/**
- * Makes the mock server respond with different (other) Alertmanager config
- */
-export const setAlertmanagerConfig = (config: AlertManagerCortexConfig) => {
```

Comment:
This kind of logic is instead in the "mock entities" rather than the server behaviour. It's kind of a valid distinction - this file should probably be more for changing how endpoint behave (i.e. what status code do you get, is there an error), and the entities are more like "what if the AM config looks like this"

I think it'll evolve over time and we can discover the right balance